scripts/do_submission.sh

```diff
@@ -10,7 +10,13 @@ set -e
 WID=$(xdotool search --all --onlyvisible --desktop 1 --class "google-chrome")
 CMD="xdotool windowactivate --sync ${WID}"
 WAIT=0.5
-$CMD mousemove 1816 324 click 1
+
+$CMD key F5
+sleep 3
```

Comment:
[Bare sleeps](https://www.martinfowler.com/articles/nonDeterminism.html#AsynchronousBehavior) are not a robust solution to asynchronous behavior, why is this script not using the same tools as `test:browser` or some standard browser automation with bindings for a programming language?

Reply:
Answered in person: exploratory testing usage rather than automated testing, usage from a laptop against both development and staging